tests/cpp/test_multidevice_transformer.cpp

```diff
@@ -548,6 +548,111 @@ TEST_P(DistributedTransformerTest, MLP_Backward) {
   validate(expected_outputs, outputs);
 }
 
+TEST_P(DistributedTransformerTest, Forward) {
+  auto dtype = DataType::Half;
+  at::ScalarType at_dtype = data_type_to_aten(dtype);
+  auto fusion = std::make_unique<Fusion>();
+  FusionGuard fg(fusion.get());
+  const auto mesh = DeviceMesh::createForNumDevices(D);
+
+  TensorView* x = makeContigConcreteTensor({B * S, E}, DataType::Float);
+  TensorView* mha_w0 = makeContigConcreteTensor({D, E, 3 * E / D}, dtype);
+  TensorView* mha_b0 = makeContigConcreteTensor({D, 3 * E / D}, dtype);
+  TensorView* mha_w1 = makeContigConcreteTensor({D, E / D, E}, dtype);
+  TensorView* mha_b1 = makeContigConcreteTensor({E}, dtype);
+  TensorView* mlp_w0 = makeContigTensor(3, dtype);
+  TensorView* mlp_b0 = makeContigTensor(2, dtype);
+  TensorView* mlp_w1 = makeContigTensor(3, dtype);
+  TensorView* mlp_b1 = makeContigTensor(1, dtype);
+
+  fusion->addInput(x);
+  fusion->addInput(mha_w0);
+  fusion->addInput(mha_b0);
+  fusion->addInput(mha_w1);
+  fusion->addInput(mha_b1);
+  fusion->addInput(mlp_w0);
+  fusion->addInput(mlp_b0);
+  fusion->addInput(mlp_w1);
+  fusion->addInput(mlp_b1);
+
+  constexpr float kEps = 1e-5;
+  Val* eps_ptr = IrBuilder::create<Val>(kEps);
+  std::vector<int64_t> norm_shape{E};
+
+  auto ln_1 = layer_norm(x, norm_shape, nullptr, nullptr, eps_ptr);
```

Comment:
```suggestion
  auto ln_1 = layer_norm(x, norm_shape, /*weight=*/nullptr, /*bias=*/nullptr, eps_ptr);
```
so it got checked by https://clang.llvm.org/extra/clang-tidy/checks/bugprone/argument-comment.html